rust/relay/ebpf-turn-router/src/lib.rs

```diff
@@ -0,0 +1 @@
+#![cfg(target_os = "linux")]
```

Comment:
What if we cfg the entire `main.rs` on Linux only? Does that work? I am happy to keep a dummy lib.rs for all other platforms.

Reply:
that could work too! I'll try this

Reply:
worked